glean-core/src/lib.rs

```diff
@@ -102,7 +102,9 @@ impl Glean {
     }
 
     pub fn snapshot(&mut self, store_name: &str, clear_store: bool) -> String {
-        StorageManager.snapshot(&self.storage(), store_name, clear_store)
+        StorageManager
```

Comment:
Can we add unit test coverage to make sure that `snapshot` behaves as expected?

Reply:
We have a couple of tests using the snapshot function in tests/simple.rs.
We need to improve our testing strategy internally anyway. Ok if I take this in a followup?

Reply:
Ok!